_source/_posts/2023-06-30-podcast-approaching-identity-automation.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: blog_post
+title: "June Podcast: Approaching Automation with Jeff Taylor"
+author: edunham
+by: advocate
+communities: [devops,security]
+description: "Listen and learn about the automation journey"
+tags: [podcast, automation, enterprise]
+image: blog/identity-developer-podcast/podcast.jpg
+type: awareness
+---
+
+# Approaching Automation
+
+Product Manager Jeff Taylor joins us for the first time to discuss the principles behind successful automation. 
+
+What's the automation journey, and how does it relate to organizational maturity? How can you scale your work with automation while avoiding the pitfalls of premature optimization? 
+
+If you're ready to start automating Okta administration tasks, where should you start? 
+
+## Episode Links
+
+{% youtube LilghHdW18Q %} 
+
+Listen to the audio version on Casted [here](https://listen.casted.us/public/49/Workforce-Identity-Developer-Podcast-4ce90a5f/59f5cd1e). 
+
+Where are you in your own automation journey? What are your next steps to enhance automation? Let us know in the comments below!
```

Comment:
Probably should add the CTA to follow OktaDev on Twitter and subscribe on YouTube (can they subscribe on Casted??) to get notified about the next episode or other exciting content.

Also do you have resources that people may want to read more about?

Reply:
@edunham I think adding the same (or similar) CTA you have for July podcast here will be 👍🏼 . Then let's ship it. :shipit:

Reply:
twitter/youtube added here, cta PR'd for July :)